pkg/mount/block/blockdev.go

```diff
@@ -74,6 +74,60 @@ func (b BlockDev) Mount(path string, flags uintptr) (*mount.MountPoint, error) {
 	return mount.TryMount(devpath, path, flags)
 }
 
+// GPTTable tries to read a GPT table from the block device described by the
+// passed BlockDev object, and returns a gpt.Table object, or an error if any
+func (b *BlockDev) GPTTable() (*gpt.Table, error) {
+	fd, err := os.Open(b.DevicePath())
+	if err != nil {
+		return nil, err
+	}
+	defer fd.Close()
+
+	blkSize, err := b.BlockSize()
```

Comment:
IIRC the block size is basically 512 for old disks and 4k for newer ones. Do we want to add an additional fallback in case the kernel doesn't return it?

Reply:
yeah, let's do that.